examples/xmlrpc/xmlrpc_main.c

```diff
@@ -88,8 +87,14 @@
  * Private Data
  ****************************************************************************/
 
-static const char *notimplemented = { "HTTP/1.1 501 Not Implemented\n\n" };
-static const char *separator = { "\015\012\015\012" };
+static const char *notimplemented =
+{
+  "HTTP/1.1 501 Not Implemented\n\n"
+};
+static const char *separator =
```

Comment:
Not related to this PR, but will leave here for history
```suggestion
static const FAR char *separator =
```